src/index.ts

```diff
@@ -52,7 +52,7 @@ export default {
     // Searching using github api might not return all users so I'm sticking to the db
     // this can be worked around if they change their registered wallet address
     // but it's inherently abuse-proof as to get a subsidy you must have a permit == need a contribution for 0.0003 eth/xdai
-    const { data: permits } = await supabase.from("permits").select("bounty_hunter_address").eq("bounty_hunter_address", ethAddress);
+    const { data: permits } = await supabase.from("permits").select("users(wallets(address))").eq("users.wallets.address", ethAddress);
```

Comment:
For any address this line returns:
```
[ { users: { wallets: null } } ]
``` 

Hence [here](https://github.com/ubiquity/faucet/pull/4/files#diff-a2a171449d862fe29692ce031981047d7ab755ae7f84c707aef80701b3ea0c80R58) error is shown all the time even if there are no permits for the provided address

Expected behaviour:
1. If there are no permits then allow the faucet to send a payment
2. If there are permits for the provided address then throw an error

Reply:
Thnaks, good catch. Should be sorted out now. Also took the liberty of adding `prettier` if that's ok.

Reply:
The issue remains

Do you have access to pavlovcik's supabase instance?

There is a single permit with id 1 for `beneficiary_id = 119157793` which has wallet address `0x9e4ef4353c928cd3eb473e8f12aecf58c208ef40`

So when I call the faucet for the `0x9e4ef4353c928cd3eb473e8f12aecf58c208ef40` address [this](https://github.com/ubiquity/faucet/pull/4/files#diff-a2a171449d862fe29692ce031981047d7ab755ae7f84c707aef80701b3ea0c80R83) error is expected but it is not caught right now since [permits](https://github.com/ubiquity/faucet/pull/4/files#diff-a2a171449d862fe29692ce031981047d7ab755ae7f84c707aef80701b3ea0c80R78) is `null` anyway

Reply:
> Do you have access to pavlovcik's supabase instance?

Yes

Reply:
@rndquu apologies I misunderstood the use case you mentioned before, I would believe that it is now properly fixed.

- no matching wallet / address / user, would return no result (empty list)
- matching would return a permit list